### DISCUSSION TITLE
【请教】小白请教关于 MatrixIndex 的问题

### DISCUSSION THREAD

**Discussion by leohandsome1688** [Tencent/libpag#2159] [Q&A] [answered]
export const enum MatrixIndex {
  a,
  c,
  tx,
  b,
  d,
  ty,
}

这里面的abcd都分别代表什么意思呀？getBounds和getTotalMatrix这2个有什么关联吗？

**Comment by zhalice2011** [comment, marked as answer]
你好,  matrix相关的概念可以参考 ：https://developer.mozilla.org/zh-CN/docs/Web/CSS/transform-function/matrix

getBounds 返回结果是原始边界框。 getTotalMatrix 返回结果表示图层的最终变换矩阵。包含了所有的变换信息，如平移、缩放、旋转和倾斜，这些变换将应用于图层的原始边界框以确定图层在屏幕上的最终显示位置和形状。
